ui/packages/catalog-ui-search/src/main/webapp/component/content/alert/content.alert.less

```diff
@@ -4,6 +4,6 @@
   }
 
   > .content-content {
-    height: 100%;
+    height: ~'calc(100% - 1*@{minimumButtonSize})';
```

Comment:
:question: What does this `1*` do in all of these `.less` files?

Reply:
nothing - removed them all except for the right-navigation width one because that will increase as more buttons are added